code/menuui/optionsmenu.cpp

```diff
@@ -542,6 +542,11 @@ void options_tab_setup(int  /*set_palette*/)
 			Options_sliders[gr_screen.res][i].slider.enable();
 			Options_sliders[gr_screen.res][i].slider.unhide();
 		}		
+		if (Cmdline_deadzone >= 0){
```

Comment:
Nitpick, but a space between `)` and `{` would be good here and in the following two spots.

Reply:
The bracing style in this file is inconsistent, but I agree with wookiejedi here.